Cloudinary/Shared/Coudinary.NetCoreShared/Actions/ContextParams.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Runtime.Serialization;
+
+namespace CloudinaryDotNet.Actions
+{
+    /// <summary>
+    /// Parameters of tag management
+    /// </summary>
+    public class ContextParams : BaseParams
+    {
+        List<string> m_publicIds = new List<string>();
+
+        /// <summary>
+        /// A list of Public IDs of images uploaded to Cloudinary.
+        /// </summary>
+        public List<string> PublicIds
+        {
+            get { return m_publicIds; }
+            set { m_publicIds = value; }
+        }
+
+        /// <summary>
+        /// The context name to assign or remove.
+        /// </summary>
+        public string Context { get; set; }
+
+        /// <summary>
+        /// Type
+        /// </summary>
+        public string Type { get; set; }
+
+        /// <summary>
+        /// The action to perform on image resources using the given tag.
+        /// </summary>
+        public ContextCommand Command { get; set; }
+
+        /// <summary>
+        /// Validate object model
+        /// </summary>
+        public override void Check()
```

Comment:
This method exists in each param class and is used for parameters validation before sending parameters to server. In base class this method checks some basic parameters. This method is always present in parameter class, even if the method does nothing (some param classes do not require validation). It was like this initially and we decided not to change code's style.